### DISCUSSION TITLE
Render Theme Symbol size

### DISCUSSION THREAD

**Discussion by johnkeel-thork** [mapsforge/vtm#999] [Q&A] [answered]
Is there any way to increase the size of the symbol elements in the render theme?
I can see there is a method for IRenderTheme scaleTextSize but i cant find anything for the actual symbols.
Anything that can be changed on the xml theme file or by a method call?

**Comment by devemux86** [comment, marked as answer]
`CanvasAdapter.symbolScale` and `CanvasAdapter.textScale`.
